cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -275,6 +275,16 @@ impl pallet_assets::Config<TrustBackedAssetsInstance> for Runtime {
 	type BenchmarkHelper = ();
 }
 
+// Freezer for "Trust Backed" assets instance. This enables other pallets to
+// use this pallet to set freezes and thaws on balances over accounts on a given asset.
+pub type TrustBackedAssetsFreezerInstance = pallet_assets_freezer::Instance1;
+impl pallet_assets_freezer::Config<TrustBackedAssetsFreezerInstance> for Runtime {
+	type FreezeIdentifier = ();
+	type RuntimeFreezeReason = RuntimeFreezeReason;
+	type RuntimeEvent = RuntimeEvent;
+	type MaxFreezes = ConstU32<8>;
+}
```

Comment:
```suggestion
/// Allows Freezes for the `PoolAssets` pallet.
pub type PoolAssetsFreezerInstance = pallet_assets_freezer::Instance1;
impl pallet_assets_freezer::Config<PoolAssetsFreezerInstance> for Runtime {
	type FreezeIdentifier = ();
	type RuntimeFreezeReason = RuntimeFreezeReason;
	type RuntimeEvent = RuntimeEvent;
	type MaxFreezes = ConstU32<8>;
}
```

I think we need one instance of the freezer for every instance of `pallet-asstes`. Don't forget also to set `type Freezer = {freezer_pallet}` on each `pallet-assets` instance config.